puppet/modules/web/manifests/vhost/stagingdeb.pp

```diff
@@ -18,8 +18,11 @@
   if $setup_receiver {
     secure_ssh::rsync::receiver_setup { $user:
       user           => $user,
+      homedir        => $home,
       foreman_search => 'host.hostgroup = Debian and (name = external_ip4 or name = external_ip6)',
       script_content => template('freight/rsync.erb'),
     }
+
+    File <| title == $home |> { mode => '0750' }
```

Comment:
Should `secure_ssh::receiver_setup` gain a `Stdlib::Filemode $homedir_mode = '0700'` parameter and then override it in this file?

Reply:
Right, it's our module anyways. Good idea.